cmd/config/parse.go

```diff
@@ -28,6 +30,16 @@ type OneBuildConfiguration struct {
 	Commands []map[string]string `yaml:"commands"`
 }
 
+// GetConfigFile returns the 1build configuration file from root file flag or global file variable
+func GetConfigFile() string {
```

Comment:
Why function is public? could it be private?

Reply:
Good catch, I think I was thinking it would be used outside of `config`. But currently that's not the case :).